JULIERAJ/KIDS-FIRST-Front
# Fix: Verify Email Page Text Layout #66

## Fix: Verify Email Page Text Layout #66 


## Requirements
- Text and styling should match the text in Figma
- Use svg for the icon
- Confirm the number of times user can use resend verification link
- Be creative with the coding solutions :)

## Notes
Responsiveness is not in the scope of this ticket.

## Test Steps
1. Open the application.
2. Go to Register page (Sign up)
3. Register
4. Check Sign up page according to figma (resend link is disable for 2 minutes)
5. Wait 2 minutes and click "Resend verification link" )
6. Check Modal window according to figma 
7. Close Modal window. Resend link is not available any more.
8. In case of any errors check Error page (no design)

## UI/UX Outcome
- link is disable
![Screenshot 2024-08-12 153600](https://github.com/user-attachments/assets/b0444076-aa08-4d24-9610-2b7b8ac4078e)
- link is enable
![Screenshot 2024-08-12 153824](https://github.com/user-attachments/assets/a681b911-a563-4925-9817-42b9b6b802b1)
- modal window
![Screenshot 2024-08-12 153836](https://github.com/user-attachments/assets/c01545bb-7d18-4cfc-8679-0a15f6077814)
- modal window is closed
![Screenshot 2024-08-01 145451](https://github.com/user-attachments/assets/b01a0d79-27cb-40ef-83d7-80df72a5f66f)
- Error page
![Screenshot 2024-08-12 150541](https://github.com/user-attachments/assets/5fa507a2-43ab-4628-9cb4-05789c766bac)


## Checklist
